src/LoginPage.cpp

```diff
@@ -175,7 +176,9 @@ LoginPage::LoginPage(QWidget *parent)
 
         connect(this, &LoginPage::versionOkCb, this, &LoginPage::versionOk);
         connect(this, &LoginPage::versionErrorCb, this, &LoginPage::versionError);
-        connect(this, &LoginPage::loginErrorCb, this, &LoginPage::loginError);
+        // unused thing, but if enable, you should check loginError function now,
+        // because loginError now accept QLabel *label
+        // connect(this, &LoginPage::loginErrorCb, this, &LoginPage::loginError);
```

Comment:
If this is unused, could you remive it (as well as the loginErrorCb)?

Reply:
Yep